components/brave_rewards/resources/BUILD.gn

```diff
@@ -7,16 +7,25 @@ import("//tools/grit/repack.gni")
 pack_web_resources("ui_generated_resources") {
   resource_name = "brave_rewards"
   output_dir = "$root_gen_dir/brave/components/brave_rewards/resources"
-  # need wepack and GRD generation first
+  # need webpack and GRD generation first
   deps = [
     "ui"
   ]
 }
 
+pack_web_resources("internals_generated_resources") {
```

Comment:
does this really need to be separate from the rest of the UI resources? Seems to just complicate things. Is there any benefit?

Reply:
I think I was just following the existing example in rewards (I'll have to check), but I can move it.

Reply:
if there is no specific reason then I don't think we should add another pak file here. They are generally per-component, not per-webui

Reply:
Sure, I'll eliminate it.